bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookKeeperServerStats.java

```diff
@@ -69,6 +69,11 @@
     String WATCHER_SCOPE = "bookie_watcher";
     String REPLACE_BOOKIE_TIME = "REPLACE_BOOKIE_TIME";
     String NEW_ENSEMBLE_TIME = "NEW_ENSEMBLE_TIME";
+    String FAILED_TO_RESOLVE_NETWORK_LOCATION_COUNTER = "FAILED_TO_RESOLVE_NETWORK_LOCATION_COUNTER";
+    String NEW_ENSEMBLE_NOT_ADHERING_TO_PLACEMENT_POLICY_COUNTER =
+            "NEW_ENSEMBLE_NOT_ADHERING_TO_PLACEMENT_POLICY_COUNTER";
+    String REPLACE_BOOKIE_NOT_ADHERING_TO_PLACEMENT_POLICY_COUNTER =
+            "REPLACE_BOOKIE_NOT_ADHERING_TO_PLACEMENT_POLICY_COUNTER";
```

Comment:
What is the benefit of tracking these things separately? Do you plan to act differently? If not, I would make it one instead of cluttering further.

Reply:
will make it one